CHANGELOG.unreleased.md

```diff
@@ -13,7 +13,9 @@
 - [Health] Show duplicated MAC addresses with their VIFs, VMs and networks [#5448](https://github.com/vatesfr/xen-orchestra/issues/5448) (PR [#5468](https://github.com/vatesfr/xen-orchestra/pull/5468))
 - [Host/stats] Show interfaces' names in graph "Network throughput" instead of PIFs' indices (PR [#5483](https://github.com/vatesfr/xen-orchestra/pull/5483))
 - [Pool/advanced] Ability to define default migration network [#3788](https://github.com/vatesfr/xen-orchestra/issues/3788#issuecomment-743207834) (PR [#5465](https://github.com/vatesfr/xen-orchestra/pull/5465))
-- [Proxy] Ability to run metadata backups (PR [#5499](https://github.com/vatesfr/xen-orchestra/pull/5499))
+- [Proxy] Metadata backups
+  - run backups (PR [#5499](https://github.com/vatesfr/xen-orchestra/pull/5499))
+  - list backups (PR [#5517](https://github.com/vatesfr/xen-orchestra/pull/5517))
```

Comment:
I'm not sure it's necessary to go into that much details, simply write a single phrase and link both PRs at the end ([example](https://github.com/vatesfr/xen-orchestra/blob/master/CHANGELOG.md#highlights-1)).